Jenkinsfile-datastax

```diff
@@ -274,12 +274,8 @@ pipeline {
                 '3.11',      // Previous Apache CassandraⓇ
                 '4.0',       // Previous Apache CassandraⓇ
                 '4.1',       // Current Apache CassandraⓇ
-                '5.0-beta1',    // Development Apache CassandraⓇ
-                'dse-4.8.16',   // Previous EOSL DataStax Enterprise
-                'dse-5.0.15',   // Long Term Support DataStax Enterprise
+                '5.0',       // Development Apache CassandraⓇ
                 'dse-5.1.35',   // Legacy DataStax Enterprise
-                'dse-6.0.18',   // Previous DataStax Enterprise
-                'dse-6.7.17',   // Previous DataStax Enterprise
```

Comment:
This might be too aggressive.  The top-level readme for the driver says that we support C* 2.1 and up as well as DSE 4.8.x and up.  The four releases above are now EOL and we did decide that we were going to follow the DataStax drivers policy of only supporting releases that weren't EOL at time of release... so I decided to yank these.

At a minimum we should probably update the readme to be clearer about our policy if we choose to move forward with these.

Reply:
I guess we do not need to explicitly mention policy, but update versions in README would be enough.

Reply:
The alternative that I'm considering is that we can just update 5.0 and remove the non-EOL Apache Cassandra versions now; those are properly covered by the ASF CI and this is just a "backup" for that.  Neither of those changes require an update to the support policy.

We can change the DSE versions later after we have some consensus around the approach the project will take re: backend support.